src/exposition/http.rs

```diff
@@ -256,3 +347,63 @@ mod handlers {
         }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+pub struct Metadata {
```

Comment:
These structs can probably be moved either a metriken crate? They will be shared with rpc-perf. A lot of the exposition code should also move into a common location.